mne/viz/_3d.py

```diff
@@ -1038,10 +1044,27 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
     del dig
     for key, picks in other_picks.items():
         if other_bools[key] and len(picks):
-            other_loc[key] = np.array([info['chs'][pick]['loc'][:3]
-                                       for pick in picks])
-            logger.info('Plotting %d %s location%s'
-                        % (len(other_loc[key]), key, _pl(other_loc[key])))
+            if key == 'fnirs':
+                if 'channels' in fnirs:
+                    other_loc[key] = np.array([info['chs'][pick]['loc'][:3]
+                                               for pick in picks])
+                if 'sources' in fnirs:
+                    other_loc['source'] = np.array(
+                        [info['chs'][pick]['loc'][3:6]
+                         for pick in picks])
+                    logger.info('Plotting %d %s source%s'
+                                % (len(other_loc['source']),
+                                   key, _pl(other_loc['source'])))
+                if 'detectors' in fnirs:
+                    other_loc['detector'] = np.array(
+                        [info['chs'][pick]['loc'][6:9]
+                         for pick in picks])
+                    logger.info('Plotting %d %s detector%s'
+                                % (len(other_loc['detector']),
+                                   key, _pl(other_loc['detector'])))
+                other_keys = sorted(other_loc.keys())
+                logger.info('Plotting %d %s location%s'
+                            % (len(other_loc[key]), key, _pl(other_loc[key])))
```

Comment:
This still has not been unindented

Reply:
(GitHub tip: after pushing some commits, you can view the diff yourself to quickly make sure that everything is as it should be before requesting review. I usually do this when making a PR just to make sure there isn't extraneous stuff in there or things that I missed. But even with that sometimes I miss stuff, so no big deal...)

Reply:
now it has sorry, forgot to save file